src/commands/convert/sonarqube2hdf.ts

```diff
@@ -4,23 +4,25 @@ import {SonarQubeResults as Mapper} from '@mitre/hdf-converters'
 import {checkSuffix} from '../../utils/global'
 
 export default class Sonarqube2HDF extends Command {
-  static usage = 'convert sonarqube2hdf -n <sonar_project_key> -u <http://your.sonar.instance:9000> --auth <your-sonar-api-key> -o <hdf-scan-results-json>'
+  static usage = 'convert sonarqube2hdf -n <sonar_project_key> -u <http://your.sonar.instance:9000> --auth <your-sonar-api-key> [ -b <target_branch> || -p <pull-request-id> ] -o <hdf-scan-results-json>'
 
-  static description = 'Pull SonarQube vulnerabilities for the specified project name from an API and convert into a Heimdall Data Format JSON file'
+  static description = 'Pull SonarQube vulnerabilities for the specified project name and optional branch name from an API and convert into a Heimdall Data Format JSON file'
 
-  static examples = ['saf convert sonarqube2hdf -n sonar_project_key -u http://sonar:9000 --auth YOUR_API_KEY -o scan_results.json']
+  static examples = ['saf convert sonarqube2hdf -n sonar_project_key -u http://sonar:9000 --auth YOUR_API_KEY [ -b my_branch || -p 123 ]-o scan_results.json']
 
   static flags = {
     help: Flags.help({char: 'h'}),
     auth: Flags.string({char: 'a', required: true}),
     projectKey: Flags.string({char: 'n', required: true}),
     url: Flags.string({char: 'u', required: true}),
+    branch: Flags.string({char: 'b', required: false, exclusive: ['pullRequestID'], description: "Requires Sonarqube Developer Edition or above"}),
```

Comment:
Are these two flags really exclusive?  I think the mapper can support sending both but it would be good to double check.  And presumably you can have multiple PRs targeting the same branch.  Does sonarqube not support using both of these parameters at the same time?

Reply:
while testing it out, i found combining multiple options for PR and branch did not produce any desirable outcome. while combining PR and branch, it returned no issues even when there were issues on the PR. while combining multiple PRs or multiple branches, the results seemed to be concatenated. since it did not produce desired results, i made them exclusive to avoid users inadvertently using both.

Reply:
sounds good then